providers/catalog.go

```diff
@@ -49,6 +50,26 @@ const (
 // ================================================================================
 
 var catalog = CatalogType{ // nolint:gochecknoglobals
+	Mock: {
+		AuthType: None,
+		BaseURL:  "https://not-a-real-domain.mock",
+		Support: Support{
+			BulkWrite: BulkWriteSupport{
```

Comment:
doesn't look like you've actually implemented the BulkWrite and BulkDelete methods for the mock connector, so can you set these to `false` instead? (Lines 58 to 61)

Reply:
ah right, I didn't see a dedicated interface for that so I wasn't sure how to approach it. Yeah I'll set to false.